docs/source/users/configuration.rst

```diff
@@ -195,6 +195,28 @@ Contains all modifications for pydantic `BaseModel`.
    Define the signature prefix for pydantic models.
 
 
+.. config_description:: autopydantic_model
+   :title: Show Erdantic figure
+   :path: target.configuration.ModelErdanticFigure
+   :confpy: autodoc_pydantic_model_erdantic_figure
+   :directive_option: model-erdantic-figure
+   :values: True, False
+
+   Show the entity relationship diagram of the schemas using erdantic.
+
+
+.. config_description:: autopydantic_model
+   :title: Show Erdantic figure collapsed
+   :path: target.configuration.ModelErdanticFigure
+   :confpy: autodoc_pydantic_model_erdantic_figure_collapsed
+   :directive_option: model-erdantic-figure-collapsed
+   :enable: model-erdantic-figure
+   :values: True, False
+
+   Show the entity relationship diagram collapsed or not.
+   :ref:`model-erdantic-figure <autodoc_pydantic_model_erdantic_figure>` must be True for this to have any effect.
+
```

Comment:
I feel we should also include a short warning here about the `graphviz` dependency. You've added one in the user's installation section, but most will probably won't read it anyway and rather come here to understand how to enable the feature.

Reply:
implemented in 803e7c3